core/domain/user_services_test.py

```diff
@@ -770,3 +770,129 @@ def utcnow(cls):
                 user_services.get_user_settings(self.viewer_id).last_logged_in,
                 previous_last_logged_in_datetime)
             self.logout()
+
+
+class LastExplorationEditedIntegrationTests(test_utils.GenericTestBase):
+    EXP_ID = 'exp'
+
+    def setUp(self):
+        """Create users for creating and editing exploration."""
+        super(LastExplorationEditedIntegrationTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
+        self.editor_id = self.get_user_id_from_email(self.EDITOR_EMAIL)
+
+        self.save_new_valid_exploration(
+            self.EXP_ID, self.owner_id, end_state_name='End')
+
+    def edit_exploration(self):
+        """Edit an existing exploration as EDITOR."""
+        self.login(self.EDITOR_EMAIL)
+        exp_services.update_exploration(self.editor_id, self.EXP_ID, [{
+            'cmd': 'edit_exploration_property',
+            'property_name': 'objective',
+            'new_value': 'the objective'
+        }], 'Test edit')
+        self.logout()
+
+    def reduce_last_edited_an_exploration(self, delta_hours):
```

Comment:
`_decrease_last_...`

(I thought of MapReduce when I saw 'reduce')

EDIT: consider removing this method altogether and adding it inline; see comment below.